data/categories/vp_vehicles_parts.yml

```
@@ -3940,6 +3941,35 @@
   - pattern
   - power_source
   - vehicle_type
+- id: vp-1-5-7-11
+  name: Vehicles Lifts & Stands
```

Vehicle Lifts & Stands
```suggestion
  name: Vehicle Lifts & Stands
```